react/src/lib/components/DeckGLMap/layers/axes2d/axes2DLayer.ts

```diff
@@ -1,134 +1,256 @@
+        E: { x: 4 * font_width, y: 0, width: font_width },
+        F: { x: 5 * font_width, y: 0, width: font_width },
+        G: { x: 6 * font_width, y: 0, width: font_width },
+        H: { x: 7 * font_width, y: 0, width: font_width },
+        I: { x: 8 * font_width, y: 0, width: font_width },
+        J: { x: 9 * font_width, y: 0, width: font_width },
+        K: { x: 10 * font_width, y: 0, width: font_width },
+        L: { x: 11 * font_width, y: 0, width: font_width },
+        M: { x: 12 * font_width, y: 0, width: font_width },
+        N: { x: 13 * font_width, y: 0, width: font_width },
+        O: { x: 14 * font_width, y: 0, width: font_width },
+        P: { x: 15 * font_width, y: 0, width: font_width },
+        Q: { x: 16 * font_width, y: 0, width: font_width },
+        R: { x: 17 * font_width, y: 0, width: font_width },
+        S: { x: 18 * font_width, y: 0, width: font_width },
+        T: { x: 19 * font_width, y: 0, width: font_width },
+
+        U: { x: 0, y: yh, width: font_width },
+        V: { x: font_width, y: yh, width: font_width },
+        W: { x: 2 * font_width, y: yh, width: font_width },
+        X: { x: 3 * font_width, y: yh, width: font_width },
+        Y: { x: 4 * font_width, y: yh, width: font_width },
+        Z: { x: 5 * font_width, y: yh, width: font_width },
+        0: { x: 6 * font_width, y: yh, width: font_width },
+        1: { x: 7 * font_width, y: yh, width: font_width },
+        2: { x: 8 * font_width, y: yh, width: font_width },
+        3: { x: 9 * font_width, y: yh, width: font_width },
+        4: { x: 10 * font_width, y: yh, width: font_width },
+        5: { x: 11 * font_width, y: yh, width: font_width },
+        6: { x: 12 * font_width, y: yh, width: font_width },
+        7: { x: 13 * font_width, y: yh, width: font_width },
+        8: { x: 14 * font_width, y: yh, width: font_width },
+        9: { x: 15 * font_width, y: yh, width: font_width },
+        "+": { x: 16 * font_width, y: yh, width: font_width },
+        "-": { x: 17 * font_width, y: yh, width: font_width },
+        ".": { x: 18 * font_width, y: yh, width: font_width },
+        ",": { x: 19 * font_width, y: yh, width: font_width },
+    },
+};
 
-    getAnchor(d: TextLayerData): string {
-        const is_xaxis = d.from[1] !== d.to[1];
-        if (is_xaxis) {
-            return "middle";
-        }
+export default class Axes2DLayer extends Layer<Axes2DLayerProps<unknown>> {
+    initializeState(context: LayerContext): void {
+        const { gl } = context;
 
-        const screen_from = this.context.viewport.project(d.from);
-        const screen_to = this.context.viewport.project(d.to);
-        const is_labels = d.label !== "X" && d.label !== "Y" && d.label !== "Z"; // labels on axis or XYZ annotations
-        if (is_labels) {
-            if (screen_from[0] < screen_to[0]) {
-                return "start";
-            }
-        }
+        const promise = load("font-atlas.png", ImageLoader, {
```

Comment:
I don't think this will work in an npm package. I think you should put the image in a lib directory and use an import statement.